Description: Include Pango header
 This patch includes a Pango header to fix build problems.
Author: Roland Stigge <stigge@antcom.de>

--- dia-0.97.1.orig/app/app_procs.c
+++ dia-0.97.1/app/app_procs.c
@@ -47,6 +47,10 @@
 #include <pango/pangoft2.h>
 #endif
 
+#ifdef HAVE_FREETYPE
+#include <pango/pangoft2.h>
+#endif
+
 #include <libxml/parser.h>
 #include <libxml/xmlerror.h>
 

